070-Integrations/024-cloudflare-workers.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Xata with Cloudflare Workers
+navTitle: Cloudflare Workers
+keywords: ['cloudflare', 'integration']
+description: Xata with Cloudflare Workers
+slug: integrations/cloudflare
+published: true
+---
+
+[Cloudflare](https://www.cloudflare.com/en-gb/) is a content delivery and security service that optimizes website performance and mitigates security risks for sites. [Cloudflare Workers](https://workers.cloudflare.com/) allow you to run code directly on Cloudflare's global network, reducing latency and improving performance. They're ideal for tasks like customizing web content, handling API requests, and modifying web traffic, all executed closer to the user for faster response times.
+
+> You'll need a Cloudflare account to deploy your application.
+
+The completed [Cloudflare and Xata code](https://github.com/xataio/examples/tree/main/apps/getting-started-cloudflare-workers) is available in the [Xata `examples` repo](https://github.com/xataio/examples) on GitHub.
```

Comment:
Where should the code for this live in the examples repo? `getting-started-cloudflare-workers` probably doesn't make since now since it's an integration

Reply:
The code is here: https://github.com/xataio/examples/pull/554

It only has the cloudflare dependencies, it assumes the user will run `xata init`